Configuration/Geometry/README.md

```diff
@@ -41,5 +43,6 @@ Several detector combinations have been generated:
 * D46 = T15+C9+M3+I10+O3+F2
 * D47 = T15+C10+M3+I10+O3+F3
 * D48 = T16+C9+M3+I10+O3+F2
+* D49 = T16+C9+M4+I10+O4+F2
```

Comment:
T16 has just been introduced by tracker as a test. This new muon test workflow should use the already-validated T15 tracker, as it did before this PR was rebased. Our policy for test workflows is to vary one thing at a time, to avoid interference and confusion.

Reply:
I replaced T15